pkgs/development/compilers/ghc/8.2.2-binary.nix

```diff
@@ -24,30 +24,30 @@ let
 in
```

Comment:
If you like I could change line 20 at the same time, since this is going to cause a rebuild already.